cypress/integration/circlemarker.spec.js

```diff
@@ -162,4 +162,58 @@ describe('Draw Circle Marker', () => {
 
     cy.hasVertexMarkers(4);
   });
+
+  it('disable continueDrawing', () => {
+    cy.window().then(({ map }) => {
+      map.pm.setGlobalOptions({continueDrawing: false});
+    });
+
+    cy.toolbarButton('circle-marker').click();
+    cy.get(mapSelector)
+      .click(191,216);
+
+    cy.get(mapSelector)
```

Comment:
missing the test here that the circle-marker button should not have the class `active` - to actually test that the mode was disabled after creating the circle marker